TinyBite/app/main-card-detail.tsx

```diff
@@ -8,40 +8,74 @@ import { textStyles } from "@/styles/typography/textStyles";
 import { LinearGradient } from "expo-linear-gradient";
 import { useRouter } from "expo-router";
 import { StatusBar } from "expo-status-bar";
-import React from "react";
+import React, { useState } from "react";
 import {
+  Dimensions,
   Image,
   ScrollView,
   StyleSheet,
   Text,
   TouchableOpacity,
   View,
 } from "react-native";
+import Carousel from "react-native-reanimated-carousel";
 import { SafeAreaView } from "react-native-safe-area-context";
 
+const { width: SCREEN_WIDTH } = Dimensions.get("window");
+
 export default function MainCardDetailScreen() {
   const router = useRouter();
+  const [currentPage, setCurrentPage] = useState(0);
+
+  // 이미지 슬라이더용 이미지 배열 (5장)
+  const images = [
+    require("@/assets/images/mainlist/food1.jpg"),
+    require("@/assets/images/mainlist/food1.jpg"),
+    require("@/assets/images/mainlist/food1.jpg"),
+    require("@/assets/images/mainlist/food1.jpg"),
+    require("@/assets/images/mainlist/food1.jpg"),
+  ];
 
   return (
     <>
       <StatusBar style="light" />
-      <View style={styles.container}>
-        {/* 상단 이미지 */}
-        <Image
-          style={styles.heroImage}
-          source={require("@/assets/images/mainlist/food1.jpg")}
-          resizeMode="cover"
-        />
-        {/* 딤드 효과 */}
-        <LinearGradient
-          colors={["rgba(0, 0, 0, 0.3)", "rgba(0, 0, 0, 0)"]}
-          locations={[0, 0.36]}
-          style={styles.dimmedOverlay}
-        />
-
-        {/* 컨텐츠 */}
-        <View style={{ flex: 1 }}>
-          {/* 내용 */}
+
+      {/* 컨텐츠 */}
+      <View style={{ flex: 1 }}>
+        {/* 내용 */}
+        <View style={styles.container}>
+          {/* 이미지 슬라이더 */}
+          <View style={styles.swiperContainer}>
+            <Carousel
+              width={SCREEN_WIDTH}
+              height={300}
+              data={images}
+              scrollAnimationDuration={600}
+              onSnapToItem={(index) => setCurrentPage(index)}
+              renderItem={({ item, index }) => (
+                <View key={index} style={styles.slide}>
+                  <Image
+                    style={styles.heroImage}
+                    source={item}
+                    resizeMode="cover"
+                  />
+                </View>
```

Comment:
```suggestion
            <View key={index} style={styles.slide}>
              <Image
                style={styles.heroImage}
                source={item}
                resizeMode="cover"
              />
              <LinearGradient
                colors={["rgba(0, 0, 0, 0.3)", "rgba(0, 0, 0, 0)"]}
                locations={[0, 0.36]}
                style={styles.dimmedOverlay}
              />
            </View>
```
딤드 위치를 이렇게 변경하고 styles.slide에 `position: "relative"`를 추가하면,
styles.dimmedOverlay에는 `zIndex: 1`를 삭제해도 될 것 같아요 !

Reply:
수정했습니다!!!!